packages/cli/src/cmds/preview/preview.ts

```diff
@@ -180,10 +181,16 @@ export default class Preview extends YargsCommand {
     }
     const core = coreResult.value;
 
+    const skipNgrok = !(await isNgrokCheckerEnabled());
```

Comment:
What's the default value if flag undefined?

Reply:
default is on